app/subscriber/src/features/content/list-view/interfaces/IContentListFilter.ts

```diff
@@ -8,6 +8,7 @@ export interface IContentListFilter {
   hasTopic: boolean;
   includeHidden: boolean;
   contentType?: ContentTypeName;
+  contentTypes: ContentTypeName[];
```

Comment:
Replace the single value with the multi-value.  It creates maintainability issues.